src/approximate_entropy/approximate_entropy.jl

```diff
@@ -0,0 +1,80 @@
+using DelayEmbeddings
+using Neighborhood
+using StatsBase
+using Distances
+
+export approx_entropy
+
```

Comment:
I'm confused here. Why is this like that instead of following the established API that defines some kind of entropy estimator and then calls the entropy function?

Reply:
I fixed this is the latest commit. The PR now defines the `ApproximateEntropy` estimator which works with `genentropy` (but not `probabilities`). The `approx_entropy` function is still included (as for permutation entropy, approximate entropy is widely used, so I figured a convenience function is nice to have).